src/App.tsx

```diff
@@ -14,11 +17,147 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  isStarted: boolean;
+  isReversed: boolean;
+  isDefault: boolean;
+  sortMethod: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    isStarted: false,
+    isReversed: false,
+    isDefault: true,
+    sortMethod: '',
+  };
+
+  start = () => {
+    this.setState({ isStarted: true });
+  };
+
+  reverse = () => {
+    this.setState((state) => ({
+      isReversed: !state.isReversed,
+      isDefault: false,
+    }));
+  };
+
+  sortByLength = () => {
+    this.setState(
+      {
+        sortMethod: 'byLength',
+        isDefault: false,
+      },
+    );
```

Comment:
не переноси так скобки
```suggestion
    this.setState({
      sortMethod: 'byLength',
      isDefault: false,
    });
```